rotaract-appointments.php

```diff
@@ -265,7 +279,7 @@ function appointments_settings_html() {
 	settings_errors( 'rotaract_messages' );
 	?>
 	<div class="wrap">
-		<h1><?php echo esc_html( get_admin_page_title() ); ?></h1>
+		<h1><?php esc_html_e( get_admin_page_title(), 'rotaract-appointments' ); ?></h1>
```

Comment:
As far as I know, only string literals are allowed as key parameter.
But as there are a few other lines with the same issues, I suggest we leave it and fix it later 🙈 (after discussing different solutions).